### ISSUE TITLE
Organize Import shouldn't change order of some `import` after any non-import code

### ISSUE THREAD

**Issue by Freed-Wu** [microsoft/pyright#3956]
## To Reproduce

```python
import sys
import b
sys.path.insert("src")
import a
```

Then `Organize Import by pyright`

## Actuall Behavior

```python
import sys

import a
import b
sys.path.insert("src")
```

## Expected behavior

Any non-import code can change `sys.path`, So `import a` shouldn't be changed.

```python
import sys

import b

sys.path.insert("src")
import a
```

**Comment by erictraut** [comment]
The "organize imports" feature is working as designed here. If you would like different behavior, you can use other tools like isort.

Dynamic (runtime) modification of `sys.path` is highly discouraged and won't work with static type checking in general. I recommend you use a different approach.

**Comment by trygveaa** [comment]
Is there any way to prevent lines from being moved by organize imports? E.g. a comment that could be placed on the `import b` line?

There are certain special situations where one has no other option than to modify `sys.path`, it would be nice if it was still possible to use organize imports for the rest of the imports.

**Comment by erictraut** [comment]
No, there's currently no way to do this. The "Organize Imports" command is something that we've considered removing entirely because a static type checker arguably shouldn't care about (or have anything to do with) import ordering. This is a coding style concern which is more appropriate for a linter than a type checker. You may want to investigate other tools like isort that are designed specifically for this purpose.

**Comment by trygveaa** [comment]
Alright, thanks for the fast answer. Then I'll switch to isort for this.
